### PR TITLE
low pri: gee pr_rerun command

### PR DESCRIPTION
**Pull request by jonathan-enf**
Convenience command for rerunning presubmits.

Tested: Tested for this PR.

### CODE REVIEW

**Comment by jonathan-enf** [comment]
This PR requires approval from at least one of these reviewers:

* @ccontavalli @george-enf @jonathan-enf @minor-fixes

File: scripts/gee [hunk 1]
```diff
@@ -581,11 +581,15 @@ function _startup_checks() {
...
-  SAFE_COMMANDS+=" diff gcd get_parent hello help log lsbranches make_branch pack pr_check"
-  SAFE_COMMANDS+=" pr_checkout pr_edit pr_list pr_make pr_push pr_rollback pr_submit pr_view"
-  SAFE_COMMANDS+=" remove_branch repair restore_all_branches rupdate share update update_all"
-  SAFE_COMMANDS+=" upgrade version whatsout "
+  local -a SAFE_COMMANDS_ARRAY=(
+     bash_setup bazelgc cleanup codeowners config create_ssh_key diagnose diff
+     gcd get_parent hello help log lsbranches make_branch pack pr_check
+     pr_checkout pr_edit pr_list pr_make pr_push pr_rerun pr_rollback pr_submit
+     pr_view remove_branch repair restore_all_branches rupdate share update
+     update_all upgrade version whatsout
```
Comment (george-enf) [inline scripts/gee:589]: I would write one per line for easier diffs.

**Review by george-enf — APPROVED** [review]

**Pull request merged**